packages/core-ethereum/src/indexer/index.ts

```diff
@@ -105,9 +105,9 @@ class Indexer implements IIndexer {
   }
 
   public async getRandomChannel(): Promise<IndexerChannel | undefined> {
-    //const HACK = 3970950 // Arbitrarily chosen block for our testnet. Total hack.
+    const HACK = 9514000 // Arbitrarily chosen block for our testnet. Total hack.
```

Comment:
Did you mean to include this?

Reply:
not willingly. I've branched from titlis

Reply:
@nionis Did you insert this?

Reply:
this was a temporary hack in titlis. If you are merging back to master, you need to remove it.

Reply:
@robertkiel no